frontend/src/components/Header/LoginDialogue.tsx

```diff
@@ -11,17 +11,42 @@ import styles from "./LoginDialogue.module.css";
 
 export default function FormDialog() {
   const [open, setOpen] = useState(false);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
 
   const handleClickOpen = () => {
     setOpen(true);
   };
-  const handleLogin = () => {
+  const handleClose = () => {
     setOpen(false);
   };
-  const handleClose = () => {
+  const handleLogin = (e: any) => {
+    e.preventDefault();
+    let opts = {
+      'email': email,
+      'password': password
+    }
+    fetch('/login', {
+      method: 'post',
```

Comment:
capitalize it so it's `'POST'`

Reply:
Also look into `async` and `await` for using `fetch`. It's way cleaner than just using `.then()` on promises because you might have callback hell.